pkg/services/services.go

```diff
@@ -34,15 +44,79 @@ func New(httpClient *resty.Client, logger *slog.Logger, config configuration.Wal
 		panic("httpClient is required")
 	}
 
+	woc := whatsonchain.New(httpClient, logger, config.Chain, config.WhatsOnChain)
+
+	rawTxResultServices := servicequeue.New[RawTxResultService]().Add(servicequeue.ServiceExecutor[RawTxResultService]{
+		Name:    "WhatsOnChain",
+		Service: woc,
+	})
```

Comment:
You can configure everything using the "constructor"

```suggestion
	rawTxResultServices := servicequeue.New[RawTxResultService](
		servicequeue.NewServiceExecutor[RawTxResultService]("WhatsOnChain", woc),
	)
```